Automattic/vip-go-mu-plugins
# Files: remove legacy code for handling media uploads

## Description

We had quite a few lines of cruft/dead code, it's time to say goodbye to it.

- This supersedes #1012 (Globally enable Photon's local filters)
- Removes VIP Legacy Filesystem implementation.

This needs quite a bit of manual testing.

## Changelog Description

### Files: remove legacy code

We've removed all legacy code related to previous iterations of media implementation.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Test uploads locally
1. Test uploads on a sandbox
1. Verify resizing works as expected.
